spec/system/pokemon_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
```

Comment:
In Shipcloud nutzen wir immer double-quotes. Ich musste mich auch erst dran gewöhnen, aber jetzt ist es eine Eindeutige Regel. 😁